apps/meteor/app/api/server/v1/dns.ts

```diff
@@ -111,20 +126,28 @@ API.v1.addRoute(
 	{
 		async post() {
 			check(
-				this.queryParams,
+				this.bodyParams,
 				Match.ObjectIncluding({
 					url: String,
 				}),
 			);
 
-			const { url } = this.queryParams;
+			const { url } = this.bodyParams;
```

Comment:
```suggestion
			const { url } = this.bodyParams;
```
This fix is a breaking change. 
Add a fallback to make it still work with queryParams, just ensure it shows a deprecation notice in that case.